x-pack/plugins/cases/public/components/all_cases/columns.tsx

```diff
@@ -322,36 +326,50 @@ interface Props {
   theCase: Case;
 }
 
+const IconWrapper = styled.span`
+  svg {
+    height: 20px !important;
+    position: relative;
+    top: 3px;
+    width: 20px !important;
+  }
+`;
 export const ExternalServiceColumn: React.FC<Props> = ({ theCase }) => {
-  const handleRenderDataToPush = useCallback(() => {
-    const lastCaseUpdate = theCase.updatedAt != null ? new Date(theCase.updatedAt) : null;
-    const lastCasePush =
-      theCase.externalService?.pushedAt != null
-        ? new Date(theCase.externalService?.pushedAt)
-        : null;
-    const hasDataToPush =
-      lastCasePush === null ||
-      (lastCasePush != null &&
-        lastCaseUpdate != null &&
-        lastCasePush.getTime() < lastCaseUpdate?.getTime());
-    return (
-      <p>
-        <EuiLink
-          data-test-subj={`case-table-column-external`}
-          href={theCase.externalService?.externalUrl}
-          target="_blank"
-          aria-label={i18n.SERVICENOW_LINK_ARIA}
-        >
-          {theCase.externalService?.externalTitle}
-        </EuiLink>
-        {hasDataToPush
-          ? renderStringField(i18n.REQUIRES_UPDATE, `case-table-column-external-requiresUpdate`)
-          : renderStringField(i18n.UP_TO_DATE, `case-table-column-external-upToDate`)}
-      </p>
-    );
-  }, [theCase]);
-  if (theCase.externalService !== null) {
-    return handleRenderDataToPush();
+  const { triggersActionsUi } = useKibana().services;
+
+  if (theCase.externalService == null) {
+    return renderStringField(i18n.NOT_PUSHED, `case-table-column-external-notPushed`);
   }
-  return renderStringField(i18n.NOT_PUSHED, `case-table-column-external-notPushed`);
+
+  const lastCaseUpdate = theCase.updatedAt != null ? new Date(theCase.updatedAt) : null;
+  const lastCasePush =
+    theCase.externalService?.pushedAt != null ? new Date(theCase.externalService?.pushedAt) : null;
+  const hasDataToPush =
+    lastCasePush === null ||
+    (lastCasePush != null &&
+      lastCaseUpdate != null &&
+      lastCasePush.getTime() < lastCaseUpdate?.getTime());
+  return (
+    <p>
+      <IconWrapper>
+        <EuiIcon
+          size="original"
+          title={theCase.externalService?.connectorName}
+          type={triggersActionsUi.actionTypeRegistry.get(theCase.connector.type)?.iconClass ?? ''}
+        />
+      </IconWrapper>
+      <EuiLink
+        data-test-subj={`case-table-column-external`}
+        title={theCase.externalService?.connectorName}
+        href={theCase.externalService?.externalUrl}
+        target="_blank"
+        aria-label={i18n.SERVICENOW_LINK_ARIA}
```

Comment:
I think the label should be generic.

Reply:
yes! i moved this, mustve been this way a long long time